runtime/static_config.go

```diff
@@ -283,27 +243,29 @@ func (conf *StaticConfig) MustGetStruct(key string, ptr interface{}) {
 // SetConfigValueOrDie sets the static config value.
 // dataType can be a boolean, number or string.
 // SetConfigValueOrDie will panic if the config is frozen.
-func (conf *StaticConfig) SetConfigValueOrDie(key string, bytes []byte, dataType string) {
+func (conf *StaticConfig) SetConfigValueOrDie(key string, str string, dataType string) {
```

Comment:
Since you are at it, the `dataType` should really be an enum type, so that there would be possible to set a invalid dataType.

Reply:
Unfortunately, there is a use case, where the type is from a config file. In that case the type is a string. If we change the type into an enum, then our customer need to parse the type. I would say let's keep this as it is for now. SetConfigValueOrDie now only accepts string "boolean", "number" and "string" as type.

Reply:
Do we have the uses case in /examples/example-gateway?

Reply:
ok, I see. We can probably deal with that in a separate PR.